DEVELOPER_GUIDE.md

```diff
@@ -24,6 +24,35 @@ Ultimately, your directory structure should look like this:
 │       └── dashboards-assistant
 ```
 
+Setup local stack
+
+1. Export keys locally:
+```sh
+# defaults to us-west-2
+export REGION=
+export AWS_ACCESS_KEY_ID=
+export AWS_SECRET_ACCESS_KEY=
+export AWS_SESSION_TOKEN=
+```
+2. Setup OSD plugins
+```sh
+# from OSD root folder
+cd plugins
+git clone https://github.com/opensearch-project/dashboards-assistant
+(cd dashboards-assistant && yarn setup)
+
+# example with security plugin
+git clone --depth 1 --branch $VERSION https://github.com/opensearch-project/security-dashboards-plugin.git
+
+cd ../../
```

Comment:
you are in `plugins` directory, `cd ../..` goes to parent of osd dir instead of osd dir

Reply:
good catch!